src/Point/rasta-point-web/src/proto/PointServiceClientPb.ts

```diff
@@ -6,8 +6,8 @@
 
 // Code generated by protoc-gen-grpc-web. DO NOT EDIT.
 // versions:
-// 	protoc-gen-grpc-web v1.5.0
-// 	protoc              v4.25.1
+// 	protoc-gen-grpc-web v1.4.2
```

Comment:
somehow I seem to use a newer version of protoc-gen-web (through brew). This leads to the diff below, where the nullable annotations are now being removed. Unfortunately, the protoc-gen-grpc-web v1.5.0 does not yet seem to be available through npm.